modules/appeals_api/config/routes.rb

```diff
@@ -215,7 +215,7 @@
       get :docs, to: '/appeals_api/docs/v2/docs#ai'
 
       namespace :schemas, controller: appealable_issues_controller_path do
-        get 'appealable-issues', action: :schema
```

Comment:
I realized that this route's name didn't really make much sense - it seems like it refers to the body that's returned, but it's for the params. I updated it to match the equivalent route for legacy appeals